apps/test/unit/lib/ui/accounts/AddPasswordFormTest.jsx

```diff
@@ -108,26 +104,20 @@ describe('AddPasswordForm', () => {
       });
       const submitButton = wrapper.find('button');
       submitButton.simulate('click');
+      await handleSubmit;
```

Comment:
Could do this same cleanup with the `describe('on successful submission'` section above.